core/templates/dev/head/pages/topics_and_skills_dashboard/topics_and_skills_dashboard.html

```diff
@@ -25,38 +25,35 @@ <h2 class="dashboard-title" translate="I18N_DASHBOARD_TOPICS_AND_SKILLS_DASHBOAR
         <li ng-if="topicSummaries.length > 0"
             ng-class="{'dashboard-tabs-active': activeTab === TAB_NAME_TOPICS}">
           <a class="dashboard-tabs-text"
-             ng-click="setActiveTab(TAB_NAME_TOPICS)"
-             translate="I18N_DASHBOARD_TOPICS">
+             ng-click="setActiveTab(TAB_NAME_TOPICS)"> Topics
           </a>
         </li>
         <li ng-if="skillSummaries.length > 0"
             ng-class="{'dashboard-tabs-active': activeTab === TAB_NAME_SKILLS}">
           <a class="dashboard-tabs-text"
-             ng-click="setActiveTab(TAB_NAME_SKILLS)"
-             translate="I18N_DASHBOARD_SKILLS">
+             ng-click="setActiveTab(TAB_NAME_SKILLS)"> Skills
           </a>
         </li>
       </ul>
 
-    <div ng-if="activeTab === TAB_NAME_TOPICS">
-      <topics-list topic-summaries="topicSummaries"></topics-list>
-    </div>
+      <div ng-if="activeTab === TAB_NAME_TOPICS">
+        <topics-list topic-summaries="topicSummaries"></topics-list>
+      </div>
 
-    <div ng-if="activeTab === TAB_NAME_SKILLS">
-      <skills-list skill-summaries="skillSummaries"></skills-list>
+      <div ng-if="activeTab === TAB_NAME_SKILLS">
+        <skills-list skill-summaries="skillSummaries"></skills-list>
+      </div>
     </div>
-  </div>
 
 
     <div class="col-sm-8" ng-if="topicSummaries.length === 0 && skillSummaries.length === 0">
       <blockquote class="intro-card">
-        <p class="intro-card-message" style="padding-top: 6px;"
-           translate="I18N_TOPICS_AND_SKILLS_DASHBOARD_INTRO_MESSAGE">
+        <p class="intro-card-message" style="padding-top: 6px;"> No topics or skills have been created yet.
```

Comment:
Ditto, move "No topics ..." to its own line, indented from the p tag.

In general, if it's not of the form `<tag>...</tag>` on a single line, have a line break after the opening tag. /cc @apb7 @kevinlee12

Reply:
Ok, done.

Reply:
Added in Issue #5108.